misc/packaging/README.rst

```diff
@@ -3,228 +3,14 @@
 ACRN Installation via Debian Packages
 #####################################
 
-Debian packages provide a simple way to build and package a collection
-of ACRN configurations on a development system, based on a set of
-hardware platforms and scenario choices.  You can then copy the packages
-onto your target platform, select a particular configuration and reboot
-the system with ACRN and an Ubuntu Service VM up and running.
+Debian packages provide a simple way to build and package ACRN configurations on
+a development computer. You can then copy the packages onto your target system,
+install the packages, and reboot the system with ACRN and an Ubuntu Service VM
+up and running.
 
-Follow these instructions to build Debian packages for the acrn-hypervisor and
-acrn-kernel, install them on your target system, and boot running ACRN.
+By default, Debian packages are created when you build your ACRN hypervisor and
+ACRN kernel via Makefile commands. See the `Getting Started Guide <https://projectacrn.github.io/latest/getting-started/getting-started.html>`_ for the build steps.
```

Comment:
```suggestion
ACRN kernel via ``Makefile`` commands. See the `Getting Started Guide <https://projectacrn.github.io/latest/getting-started/getting-started.html>`_ for the build steps.
```

Reply:
It's better to use
```
:ref:`gsg`
```
Rather than a direct URL

Reply:
Perhaps add as well that not only the build steps are in the GSG but also the installation steps.